pallets/phala/src/stakepool.rs

```diff
@@ -1106,9 +1016,16 @@ pub mod pallet {
 				pool_info.workers.contains(&worker),
 				Error::<T>::WorkerDoesNotExist
 			);
+			ensure!(
+				!pool_info.cd_workers.contains(&worker),
+				Error::<T>::WorkerExists
```

Comment:
```suggestion
				Error::<T>::WorkerAlreadyStopped
```

Reply:
done

Reply:
I meant to make this error message "WorkerAlreadyStopped", but you have just replaced all the "WorkerExists". This violates the semantic of the existing errors.

Reply:
done